src/live_restore/live_restore.h

```diff
@@ -0,0 +1,45 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+#pragma once
+
+typedef enum {
+    WT_LIVE_RESTORE_FS_LAYER_DESTINATION,
+    WT_LIVE_RESTORE_FS_LAYER_SOURCE
+} WT_LIVE_RESTORE_FS_LAYER_TYPE;
+
+/*
+ * __wt_live_restore_fs_layer --
+ *     A layer in a live restore file system.
+ */
+struct __wt_live_restore_fs_layer {
+    const char *home;
+    WT_LIVE_RESTORE_FS_LAYER_TYPE which;
+};
+
+/*
+ * __wt_live_restore_fs --
+ *     A live restore file system in the user space, which consists of one or more actual FS layers.
```

Comment:
which consists of one or more actual FS layers. -> which consists of two layer, a source and a destination.

Reply:
d2c052ff9